.github/workflows/docs.yml

```diff
@@ -24,9 +24,9 @@
 name: publish docs with release
 
 on:
-  release:
-    types:
-      - published
+  push:
+    tags:
+      - 'v?[0-9]+.[0-9]+.[0-9]+'
```

Comment:
So this isn't defined in multiple places, should this workflow just merge with the promote-release workflow?

Reply:
I had thought about doing that...didn't want to stomp on a working flow that does too many things.  

The docs should always be paired with the release as far as I can imagine.  I can move it there if you want.